### ISSUE TITLE
Hoverable card names in dialog boxes

### ISSUE THREAD

**Issue by brine** [octgn/OCTGN#1336]
This one's a little crazy, but it'd be great if we could get card names to be hoverable inside dialog boxes.  For example, the custom text you can feed into the askInteger() function:

``` python
num = askInteger("How many counters to place on {}?".format(card), 0)
```

This would allow the player to hover over the card's name inside the dialog's text, see the preview of the card's image.  Similarly to how it works in the chat log, I guess?

**Comment by kellyelton** [comment]
You're right, this is a bit crazy.

So what I'm thinking is if you pass into the string `{GUID}` that could get translated into the specific card.

But passing in `format(card)` wouldn't work.

**Comment by brine** [comment]
format(card) works for notify though

**Comment by kellyelton** [comment]
o does it? Maybe it actually passes back a guid, but then the resulting string figures it out already.

**Comment by brine** [comment]
Yeah theres stuff in chatcontrol i think, maybe we can pull it out to its own file and allow the dialog boxes to call it too?

**Comment by kellyelton** [comment]
well no, the magic happens in python

```  def __format__(self, format_spec):
    return '{#%d}' % self._id`1``
